docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+    "it is a blocking function.\n",
+    "\n",
+    "Finally, we cancel the monitor of the button."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from apstools.plans import run_blocking_function\n",
+    "from ophyd.status import Status\n",
+    "\n",
+    "def run_sscan():\n",
+    "    st = Status(timeout=20)\n",
+    "\n",
+    "    def watch_the_button(old_value, value, **kwargs):\n",
+    "        if old_value == 1 and value == 0:\n",
+    "            st.set_finished()\n",
+    "            scan_button.clear_sub(watch_the_button)\n",
+    "\n",
+    "    yield from bps.mv(scan_button, 1)\n",
+    "    scan_button.subscribe(watch_the_button)\n",
+    "    yield from run_blocking_function(st.wait)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Run preconfigured `sscan` record\n",
+    "\n",
+    "The `apstools.synApps.SscanRecord` class provides access to most of the fields\n",
+    "provided by a `sscan` record.  Use `SscanRecord` to connect and repeat the above\n",
+    "example.  With the `SscanRecord` class, the button is called `execute_scan`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from apstools.synApps import SscanRecord\n",
+    "\n",
+    "scan1 = SscanRecord(f\"{IOC}scan1\", name=\"scan1\")\n",
+    "scan1.wait_for_connection()\n",
+    "\n",
+    "def run_sscan():\n",
+    "    st = Status(timeout=20)\n",
+    "\n",
+    "    def watch_execute_scan(old_value, value, **kwargs):\n",
+    "        if old_value == 1 and value == 0:\n",
+    "            st.set_finished()\n",
+    "            scan1.execute_scan.clear_sub(watch_execute_scan)\n",
+    "\n",
+    "    yield from bps.mv(scan1.execute_scan, 1)\n",
+    "    scan1.execute_scan.subscribe(watch_execute_scan)\n",
+    "    yield from run_blocking_function(st.wait)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Retrieve the data collected by `scan1` as a dictionary of numpy arrays.  Include\n",
+    "both detectors and the motor.  The `sscan` record has buffers capable of\n",
+    "collecting as many as 1,000 points per array.  First get the number of points\n",
```

Comment:
I am not sure we need that information. Besides, this is the default but beamlines may have changed that in their IOC. At 29 we have it set to 100000.

Reply:
I included it because my naive self at first got the full arrays. A rookie mistake, but good to note.

Not considering the situation of customized IOCs, just describing how it is defined in synApps.  ophyd/pyepics will pick up whatever is the full array size.